gateway/main/urls.py

```diff
@@ -23,6 +23,9 @@
 from drf_yasg import openapi
 import probes.views
 import version.views
+from api.utils import custom_server_error
+
+handler500 = custom_server_error
```

Comment:
@paaragon I think we can use this approach from DRF I don't know if you had the opportunity to take a look:

https://www.django-rest-framework.org/api-guide/exceptions/#rest_frameworkexceptionsserver_error

It would be just to use the next line:
```
handler500 = 'rest_framework.exceptions.server_error'
```

I didn't know about it, I just saw it after reading the documentation about the `handler500` variable. And we can remove the method `custom_server_error`.